log4j/src/main/java/com/tersesystems/echopraxia/log4j/Log4JCoreLogger.java

```diff
@@ -47,7 +47,7 @@ public boolean isEnabled(Level level, Condition condition) {
 
   @Override
   public void log(Level level, String message) {
-    logger.log(convertLevel(level), createMarker(), message);
+    logger.log(convertLevel(level), createMarker(), createMessage(message));
```

Comment:
Needs to call createMessage to include a `Message` that contains arguments, it's not just the message template.